modules/3d-tiles/src/lib/parsers/helpers/parse-3d-impicit-tile.ts

```diff
@@ -128,9 +128,10 @@ export async function parseImplicitTiles(
     tile.contentUrl = replaceContentUrlTemplate(contentUrlTemplate, lev, x, y, z);
   }
 
+  const childTileLevel = level + 1;
```

Comment:
File name has a typo `parse-3d-impicit-tile.ts` => parse-3d-implicit-tile.ts` (not sure if it should be plural tiles, normally we match the file name and the main export which in this case is the function `parseImplicitTiles`

Reply:
Renamed